nishiyama/chap1/knock01.py

```diff
@@ -0,0 +1,3 @@
+word = 'パタトクカシーー'
+# print(word[0:1] + word[2:3] + word[4:5] + word[6:7])
+print(word[::2])
```

Comment:
pythonのソースコードは原則，最後の行に改行が必要です．
改行がないと，github上で改行しろというマークが出たり，vscodeでwarningが出たりするので，改行を追加しよう　
```suggestion
print(word[::2])

```